build.gradle

```diff
@@ -34,7 +34,10 @@ allprojects {
 subprojects {
   apply plugin: 'com.github.ethankhall.semantic-versioning'
   apply plugin: 'nebula.provided-base'
-  apply plugin: 'net.ltgt.errorprone'
+  if (!"9".equals("$System.env.HOME")) {
```

Comment:
Can we use [JavaVersion](https://docs.gradle.org/current/javadoc/org/gradle/api/JavaVersion.html) and keep errorprone if on JDK8?